linode/resource_linode_firewall.go

```diff
@@ -56,6 +56,38 @@ func resourceLinodeFirewallRule() *schema.Resource {
 	}
 }
 
+func resourceLinodeFirewallDevice() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeInt,
+				Description: "The ID of the firewall device.",
+				Computed:    true,
+			},
+			"entity_id": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "The ID of the underlying entity for the firewall device (e.g. the Linode's ID).",
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Description: "The type of firewall device.",
+				Computed:    true,
+			},
+			"label": {
+				Type:        schema.TypeString,
+				Description: "The label of the underlying entity for the firewall device.",
+				Computed:    true,
```

Comment:
is that really computed? If it is like that in the resource maybe it needs to be changed?

Reply:
Oh hmm, let me look into that.

Reply:
Just looked through the resource in master and everything under devices seems to be computed, so I'll assume this can stay.